doc/classes/BoneMapProfile.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="BoneMapProfile" inherits="Resource" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+	</brief_description>
+	<description>
+		This resource is used in [EditorScenePostImport]. Some parameters are referring bone in [Skeleton3D], [Skin], [Animation] and some other nodes is rewritten by depending on the parameters of [BoneMapProfile].
```

Comment:
Some language issues, not sure what the intended meaning was exactly:
```suggestion
		This resource is used in [EditorScenePostImport]. Some parameters are referring to bones in [Skeleton3D], [Skin], [Animation], and some other nodes are rewritten based on the parameters of [BoneMapProfile].
```

Reply:
My suggestion:

This resource is used in [EditorScenePostImport] to rename bones in [Skeleton3D] during scene import, as well as update their references from [Skin], [Animation], and other nodes.